examples/notebooks/get-started/learn_beam_windowing_by_doing.ipynb

```diff
@@ -0,0 +1,1880 @@
+        "<hr style=\"border: 5px solid #003262;\" />\n",
+        "\n",
+        "## Load the Data\n",
+        "\n",
+        "Let's import the example data we will be using throughout this tutorial. The [dataset](https://www.kaggle.com/datasets/fedesoriano/air-quality-data-in-india) consists of **hourly air quality data (PM 2.5) in India from November 2017 to June 2022**.\n",
+        "\n",
+        "> The World Health Organization (WHO) reports 7 million premature deaths linked to air pollution each year. In India alone, more than 90% of the country's population live in areas where air quality is below the WHO's standards.\n",
+        "\n",
+        "**What does the data look like?**\n",
+        "\n",
+        "The data set has 36,192 rows and 6 columns in total recording the following attributes:\n",
+        "\n",
+        "1.   `Timestamp`: Timestamp in the format YYYY-MM-DD HH:MM:SS\n",
+        "2.   `Year`: Year of the measure\n",
+        "3.   `Month`: Month of the measure\n",
+        "4.   `Day`: Day of the measure\n",
+        "5.   `Hour`: Hour of the measure\n",
+        "6.   `PM2.5`: Fine particulate matter air pollutant level in air\n",
+        "\n",
+        "**For our purposes, we will focus on only the first and last column of the** `air_quality` **DataFrame**:\n",
+        "\n",
+        "1.   `Timestamp`: Timestamp in the format YYYY-MM-DD HH:MM:SS\n",
+        "2.   `PM 2.5`: Fine particulate matter air pollutant level in air\n",
+        "\n",
+        "Run the following cell to load the data into our file directory."
+      ],
+      "metadata": {
+        "id": "W_63UtsoBRql"
+      }
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 3,
+      "metadata": {
+        "id": "GTteBUZ-7e2s",
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "outputId": "3af9cdb0-c248-4c6d-96f6-c3739fb66014"
+      },
+      "outputs": [
+        {
+          "output_type": "stream",
+          "name": "stdout",
+          "text": [
+            "Copying gs://batch-processing-example/air-quality-india.csv...\n",
+            "/ [1 files][  1.4 MiB/  1.4 MiB]                                                \n",
+            "Operation completed over 1 objects/1.4 MiB.                                      \n"
+          ]
+        }
+      ],
+      "source": [
+        "# Copy the dataset file into the local file system from Google Cloud Storage.\n",
+        "!mkdir -p data\n",
+        "!gsutil cp gs://batch-processing-example/air-quality-india.csv data/"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "#### Data Preparation\n",
+        "\n",
+        "Before we load the data into a Beam pipeline, let's use Pandas to select two columns."
+      ],
+      "metadata": {
+        "id": "1NcmPl7C43lY"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "# Load the data into a Python Pandas DataFrame.\n",
```

Comment:
Nice, but maybe we should use the Beam Dataframe as an example? :P

Reply:
well, Beam Dataframe already has a good notebook: https://colab.sandbox.google.com/github/apache/beam/blob/master/examples/notebooks/tour-of-beam/dataframes.ipynb. I am not sure whether it covers all the operations for this example.